carball/tests/stats/kickoff_test.py

```diff
@@ -32,4 +41,8 @@ def test(analysis: AnalysisManager):
             kickoffs = proto_game.game_stats.kickoffs
             assert (len(kickoffs) == 2)
 
+            first_player = proto_game.players[0].stats
```

Comment:
Can we test that the positioning is right? Like cheat, boost, etc. At least one replay.